src/reducers/RegisterReducer.js

```diff
@@ -0,0 +1,17 @@
+import { REGISTER_USER } from '../actions/types';
+
+const RegisterReducer = (action) => {
+  switch (action.type) {
+    // TODO Authentificated should not always be true.
+    case REGISTER_USER:
+      return {
+        ...state,
+        admin: false,
```

Comment:
admin and auth should be in auth reducer only, i think what ucla does with registered / user works